### PR TITLE
Clarified the use of the AIP client in coexistence with built-in labels (added paragraphs in the right order this time)

### PR DESCRIPTION
**Pull request by E-Saggese**
…ls (in the right order this time)

The vast majority of customers interviewed didn't know this critical scenario was an option, and as a result most of them were not using built-in labels in Office.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 1]
```diff
@@ -129,9 +129,10 @@ Deploy this setting by using Group Policy, or by using the [Office cloud policy
 
 ### Office built-in labeling client and the Azure Information Protection client
 
-If users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2), by default, the built-in labeling client is turned off in their Office apps. 
+You may want to use the built-in sensitivity labeling capabilities in Office to benefit from the latest features and enhance user experience while at the same time having the abilit to label and protect non-Office files from the desktop with the Azure Information Protection client.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
You might want to use the built-in sensitivity labeling capabilities in Office to benefit from the latest features and enhance user experience while at the same time having the ability to label and protect non-Office files from the desktop with the Azure Information Protection client. 
```

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 2]
```diff
@@ -129,9 +129,10 @@ Deploy this setting by using Group Policy, or by using the [Office cloud policy
 
 ### Office built-in labeling client and the Azure Information Protection client
 
-If users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2), by default, the built-in labeling client is turned off in their Office apps. 
+You may want to use the built-in sensitivity labeling capabilities in Office to benefit from the latest features and enhance user experience while at the same time having the abilit to label and protect non-Office files from the desktop with the Azure Information Protection client. 
+By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2) the built-in labeling client is turned off in their Office apps and the plugin functionality is used instead.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2), the built-in labeling client is turned off in their Office apps and the plug-in functionality is used instead. 
```

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 3]
```diff
@@ -129,9 +129,10 @@ Deploy this setting by using Group Policy, or by using the [Office cloud policy
...
+By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2) the built-in labeling client is turned off in their Office apps and the plugin functionality is used instead. 
 
-To use built-in labeling rather than the Azure Information Protection client for Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).
+To maintain the ability to use the Azure Information Protection clientfor labeling non-Office files from the desktop without disabling the built-in labeling functinality within Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
To maintain the ability to use the Azure Information Protection client for labeling non-Office files from the desktop without disabling the built-in labeling functionality within Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).
```

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 4]
```diff
@@ -149,6 +150,14 @@ Deploy this setting by using Group Policy, or by using the [Office cloud policy
 > [!NOTE]
 > If you use the Group Policy setting **Use the Sensitivity feature in Office to apply and view sensitivity labels** and set this to **1**, there are some situations where the Azure Information Protection client might still load in Office apps. Blocking the add-in from loading in each app prevents this happening.
 
+You can also use the following registry value to quickly toggle between using the Azure Information Protection plugin and built-in sensitivity labels in Office apps:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
You can also use the following registry value to quickly toggle between using the Azure Information Protection plug-in and built-in sensitivity labels in Office apps:
```

File: microsoft-365/compliance/sensitivity-labels-office-apps.md [hunk 5]
```diff
@@ -149,6 +150,14 @@ Deploy this setting by using Group Policy, or by using the [Office cloud policy
...
+
+|**Registry key**|**Type**|**Value** |
+|:-------|:------:|--------:|
+|HKCU\Software\Microsoft\Office\16.0\Common\Security\Labels\UseOfficeForLabelling  |REG_DWORD  |1  |
+
+Setting this value to 1 enables the built-in sensitivity labeling options in Office, setting it to 0 enables the AIP plugin if it is installed.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-office-apps.md]:
```suggestion
Setting this value to **1** enables the built-in sensitivity labeling options in Office; setting it to **0** enables the AIP plug-in if it is installed. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @E-Saggese, kindly commit the suggested changes. Many thanks.

**Comment by E-Saggese** [comment]
Hi @JohanFreelancer9, apologies for my ignorance, I committed the changes, but do I need to close the pull request? When I do it tells me the branch has uncommitted changes.

**Review by JohanFreelancer9 — APPROVED** [review]
@E-Saggese, thank you for committing the changes. No, please don't close the pull request. It needs to be approved by the document author/manager, and will then be merged.

**Comment by yogkumgit** [comment]
@cabailey Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cabailey Gentle reminder.

**Comment by cabailey** [comment]
Closing after discussion with contributor

**Pull request closed without merging**